locations/spiders/basilic_and_co.py

```
@@ -0,0 +1,37 @@
+        item["name"] = "Basilic & Co"
+        item["extras"]["cuisine"] = "pizza"
+        
+        try:
+            oh = OpeningHours()
+            opening_hours_data = json.loads(response.css("display-schedule::attr(data-information)").get())["hours"]
+            for day in opening_hours_data:
+                oh.add_ranges_from_string(day["formattedHour"].replace("h",":"), days=DAYS_FR)
+
+            item["opening_hours"] = oh
+            yield item
+        except:
```

Ruff (0.16.1)

[error] Do not use bare `except`

(E722)